CHANGELOG.md

```diff
@@ -40,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed unnecessary `sqlalchemy-redshift` dependency in `drivers-sql-amazon-redshift` extra.
 - Removed unnecessary `transformers` dependency in `drivers-prompt-huggingface` extra.
 - Removed unnecessary `huggingface-hub` dependency in `drivers-prompt-huggingface-pipeline` extra.
+- `CsvRowArtifact` now inherits from `BaseTextArtifact`.
```

Comment:
Should we also note that TextArtifact inherits from BaseTextArtifact

Reply:
Yes, updated